### ISSUE TITLE
TeachingTip: Toggle theme now button text is not completely visible in 200% text scaling mode.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/microsoft-ui-xaml#3526]
**Describe the bug**

**Steps to reproduce the bug**

Pre-requisite: 
- Change the system resolution to 1280 * 768.
- Open Ease of Access settings and change the text Scaling to 200% (Ease of Access settings -> Display -> Make Text Bigger -> Change the slider to 200% -> Apply).

1. Launch XAML Controls Gallery.
2. Activate Teaching Tip from All Controls page.
3. Activate Show Teaching Tip button present under the header "Show a non-targeted Teaching Tip with buttons"
4. Check if the "Toggle Theme Now" button is completely visible or not.

**Expected behavior**

Toggle Theme Now button should be completely visible in 200% text scaling mode.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/97657307-c6200a00-1a26-11eb-83dd-6aba7404836d.png)

**Version Info**

Xaml Controls Gallery version 1.2.16.0

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_queries/edit/21531247)

**Comment by StephenLPeters** [comment]
dupe/related to #3513

**Comment by chigy** [comment]
We will be tracking this in our [internal issue](https://microsoft.visualstudio.com/OS/_workitems/edit/33595713)

**Comment by mrlacey** [comment]
Why not keep this open and update it when the internal issue is fixed too?

**Comment by chigy** [comment]
Sure, we may need a similar top level GH item we resolve these issues duplicate of. Trying to make it clear of # of source of issues...

**Comment by mrlacey** [comment]
> Sure, we may need a similar top level GH item we resolve these issues duplicate of. Trying to make it clear of # of source of issues...

Sure. A label for issues that are also being tracked internally is one way of doing it.
By closing them they can look like they're fixed, or you're not interested in allowing the community to see progress or know when these issues are actually fixed. 
Also closing the public version of an issue that is being tracked internally also undermines the message we've heard previously that eventually all work on this repo/project will be done in the open.

**Comment by chigy** [comment]
@mrlacey , yup, thanks for catching this.

**Comment by karenbtlai** [comment]
Does not repro with updated visuals so closing: 
![image](https://user-images.githubusercontent.com/7976322/134206269-52f34db6-522a-4e83-8c70-6ffe515e670d.png)
